src/app/screens/auth/sign-in/sign-in.component.scss

```diff
@@ -0,0 +1,39 @@
+@import "/src/styles/lib/colors";
+
+
+.alert {
+  color: $red;
+  margin-top: 2px;
+}
+
+.btn-submit {
+  background-color: $earls-green;
+  color: $white;
+  margin: 10px auto 15px;
+}
+
+.content-form {
+  margin: 0 25px;
+  position: relative;
+
+  &::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 2px;
+    background-color: $silver-chalice;
+  }
+}
+
+.content-input {
+  display: flex;
+  flex-flow: column;
+  margin: 15px 0;
+}
+
+.disabled {
+  background-color: $silver-chalice;
+  cursor: default;
+}
```

Comment:
You can create an auth.component.scss file with this styles and change the stylesUrl of signUp and signIn components without duplicating code.